### PR TITLE
Implement merch announcement bar

### PR DESCRIPTION
**Pull request by yelyzavetakhokhlova**
### PR Checklist

* [x] Have you followed the guidelines in our [Dev Guide](../#readme)
* [x] Have you lint your code locally prior to submission (`npm run lint`)?
* [x] Have you run prettier (`npm run format`)?
* [x] Have you checked that app can be built (`npm run build`)?
* [x] Have you checked that the build output looks according to design (screen, mobile, desktop) in Figma (`npm run serve`)?

## Visuals

image
image

### CODE REVIEW

File: src/components/AnnouncementBar/MerchIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React, { FC } from 'react';
+
+export const MerchIcon: FC = () => {
```
Comment (ViktorSoroka07) [inline src/components/AnnouncementBar/MerchIcon.tsx]: Please store it as svg with '<name>.inline.svg` and then reference as it is component. It should work just fine. Example - 'arrow.inline.svg'

File: src/components/AnnouncementBar/AnnouncementBar.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React, { FC } from 'react';
+import { useAtom } from 'jotai/index';
```
Comment (ViktorSoroka07) [inline src/components/AnnouncementBar/AnnouncementBar.tsx]:
```suggestion
import { useAtom } from 'jotai';
```

File: src/components/Layout/Layout.tsx [hunk 3]
```diff
@@ -23,8 +26,17 @@ const snowfallProps = {
...
+  if (typeof window !== 'undefined' && window.sessionStorage) {
+    return !JSON.parse(sessionStorage.getItem(ANNOUNCEMENT_CLOSED_KEY) ?? 'false');
+  }
+
+  return false;
+};
+
 export const subscriptionFormAtom = atom({ isSubmitted: false, isAlreadySubscribed: false });
 export const watchProductOverviewAtom = atom({ isOpen: false });
+export const announcementOpenAtom = atom(getInitialAnnouncementOpenState());
```
Comment (AmsterGet) [inline src/components/Layout/Layout.tsx]: May be store the all `atom`s in a separate file?
Now it leads to a circular dependency.
cc @ViktorSoroka07

Comment (yelyzavetakhokhlova) [reply src/components/Layout/Layout.tsx]: I initially had it in `AnnouncementBar` but the build failed with `cannot access before initialization`, so I just moved it here, do you have any ideas what might fix it?

Comment (AmsterGet) [reply src/components/Layout/Layout.tsx]: My suggestion is to just create a separate file with those values and import it where needed.
In this case there will be no circular references.

Before
```
Layout -> AnnouncementBar -> Layout
```

After
```
Layout -> AnnouncementBar -> atoms.js
Layout -> atoms.js
```

**Review by AmsterGet — APPROVED** [review]

**Review by ViktorSoroka07 — APPROVED** [review]

**Pull request merged**